build/grunt.js

```diff
@@ -192,7 +192,7 @@ module.exports = function(grunt) {
       minify: {
         expand: true,
         cwd: 'build/temp/',
-        src: ['video-js.css'],
+        src: ['video-js.css', 'alt/video-js-cdn.css'],
```

Comment:
would this minify task accidentally keep the minified version into `build/temp/`?